### PR TITLE
Fix/front end linting fixes

### PR DESCRIPTION
**Pull request by reeved**
# Description
This PR adds ESLint back to the Frontend and fixes errors due to linting checks.

# Demo
Before: 
image

After: 
image

# Type of change:
* Bug fix (Non-breaking change which fixes a bug)

# Checklist:
* Have you merged main into your branch?
* Have you tested your changes to ensure it works as expected and does not break existing functionality?
* If applicable, please ensure sufficient tests are added that is related to the changes.
* Please ensure this PR has a `label`, is linked to an `issue` and is related to a `project`
* If new documentation is required for this change, have you created a new documentation issue that describes the documentation needed?

### CODE REVIEW

**Review by MKTPR — APPROVED** [review]
Looks great

**Review by MKTPR — APPROVED** [review]

File: frontend/src/Components/RolesAndRules.js [hunk 1]
```diff
@@ -95,3 +96,11 @@ const RolesAndRules = ({ userRole }) => {
 };
 
 export default RolesAndRules;
+
+RolesAndRules.propTypes = {
+    userRole: PropTypes.string,
+};
+
+RolesAndRules.defaultProps = {
+    userRole: 'Civilian',
```
Comment (parfei) [inline frontend/src/Components/RolesAndRules.js:105]: Use enum please! (It's in backend folder)

File: frontend/src/Components/TopBarGame.jsx [hunk 2]
```diff
@@ -74,3 +71,18 @@ const TopBarGame = ({ userDetails, showTimer, showRole }) => {
...
+    userDetails: PropTypes.shape({
+        first: PropTypes.string,
+        last: PropTypes.string,
+    }),
+    showTimer: PropTypes.bool,
+    showRole: PropTypes.bool,
+};
+
+TopBarGame.defaultProps = {
+    userDetails: [null, 'Civilian'],
```
Comment (parfei) [inline frontend/src/Components/TopBarGame.jsx:85]: Enum here too!

**Review by parfei — APPROVED** [review]

**Comment by parfei** [comment]
Nice :D :D :D

**Pull request merged**